File: examples-sdk/swift/609-db-vilorm-overhead-bench/main.swift
// 609-db-vilorm-overhead-bench — Swift SDK equivalent
// Compile: vil compile --from swift --input 609-db-vilorm-overhead-bench/main.swift --release

let server = VilServer(name: "overhead-bench", port: 8099)
let bench = ServiceProcess(name: "bench")
bench.endpoint(method: "GET", path: "/raw/items/:id", handler: "raw_find_by_id")
bench.endpoint(method: "GET", path: "/raw/items", handler: "raw_list")
bench.endpoint(method: "GET", path: "/raw/count", handler: "raw_count")
bench.endpoint(method: "GET", path: "/raw/cols", handler: "raw_select_cols")
bench.endpoint(method: "GET", path: "/orm/items/:id", handler: "orm_find_by_id")
bench.endpoint(method: "GET", path: "/orm/items", handler: "orm_list")
bench.endpoint(method: "GET", path: "/orm/count", handler: "orm_count")
bench.endpoint(method: "GET", path: "/orm/cols", handler: "orm_select_cols")
server.service(bench)
server.compile()

File: examples-sdk/csharp/609-db-vilorm-overhead-bench/Main.cs
// 609-db-vilorm-overhead-bench — C# SDK equivalent
// Compile: vil compile --from csharp --input 609-db-vilorm-overhead-bench/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("overhead-bench", 8099);
var bench = new ServiceProcess("bench");
bench.Endpoint("GET", "/raw/items/:id", "raw_find_by_id");
bench.Endpoint("GET", "/raw/items", "raw_list");
bench.Endpoint("GET", "/raw/count", "raw_count");
bench.Endpoint("GET", "/raw/cols", "raw_select_cols");
bench.Endpoint("GET", "/orm/items/:id", "orm_find_by_id");
bench.Endpoint("GET", "/orm/items", "orm_list");
bench.Endpoint("GET", "/orm/count", "orm_count");
bench.Endpoint("GET", "/orm/cols", "orm_select_cols");
server.Service(bench);
server.Compile();
